### ISSUE TITLE
Consensus misses last base

### ISSUE THREAD

**Issue by pbnjay** [PacificBiosciences/FALCON#559]
I believe the "sentinal" tag at [falcon.c:157](https://github.com/PacificBiosciences/FALCON/blob/master/src/c/falcon.c#L157)
should also record the previous information (p_j, p_jj, p_q_base). Otherwise the g_best_ck used on line 494 refers to the previous-to-the-last base and the entire output is missing the last consensus base.

I'd submit a PR, but the included main func doesn't compile so I couldn't test. I have made these changes in my highly modified version of the algorithm and they work as expected.

**Comment by pb-jchin** [comment]
Hi, @pbnjay,  thanks a lot. Can you just post a quick diff so I can take a look?

**Comment by pbnjay** [comment]
I don't think that PR will actually work to fix the issues, but here's some example data that shows the problem: https://gist.github.com/pbnjay/bd272d27041eb7c34789c6953acce1cf

Note all the sequences begin and end with the same k-mers, so the end should be "CCCAGAGAG" but it gives "CCCAGAGA" (missing final G)

**Comment by pb-jchin** [comment]
@pbnjay Thanks for the data.  We will look into it.
